websites/A/Averge/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Averge.EU",
+    "author": {
+      "name": "lordreider",
+      "id": "853251440763600897"
+    },
+    "category": "other",
+    "logo": "https://averge.eu/cdn/img/averge.png",
+    "thumbnail": "https://averge.eu/cdn/img/averge.png",
```

Comment:
The thumbnail should be a wide promotional image or a screenshot, not the same as the logo